src/mmw/js/src/compare/views.js

```diff
@@ -255,6 +258,88 @@ var InputsView = Marionette.LayoutView.extend({
         this.ui.tableButton.addClass('active');
         this.model.set({ mode: models.constants.TABLE });
     },
+
+    downloadCSV: function() {
+        var aoi = App.currentProject.get('area_of_interest'),
+            aoiVolumeModel = new tr55Models.AoiVolumeModel({ areaOfInterest: aoi }),
+            csvHeadings = [['scenario_name', 'runoff', 'evapotranspiration', 'infiltration',
+            'tss_load', 'tss_runoff', 'tss_loading_rate', 'tn_load', 'tn_runoff', 'tn_loading_rate',
+            'tp_load', 'tp_runoff', 'tp_loading_rate']],
+            csvData = this.model.get('scenarios')
+                .map(function(scenario) {
+                    var scenarioName = scenario.get('name'),
+                        result = scenario
```

Comment:
Consider adding a refactoring commit that takes [`getScenarioResults`](https://github.com/WikiWatershed/model-my-watershed/blob/develop/src/mmw/js/src/compare/models.js#L48) and moves it somewhere accessible here and also, if you're up for it, [here](https://github.com/WikiWatershed/model-my-watershed/blob/develop/src/mmw/js/src/compare/models.js#L145) (see #2379).